src/Microsoft.ML.Dnn/DnnRetrainTransform.cs

```diff
@@ -18,6 +18,7 @@
 using NumSharp;
 using Tensorflow;
 using static Microsoft.ML.Transforms.Dnn.DnnUtils;
```

Comment:
>Microsoft.ML.Transforms.Dnn.DnnUtils [](start = 13, length = 36)

This should be ordered alphabetically.

Reply:
VS shows a warning if I reorder. I am guessing the 'using static' derivatives should appear at the end, and they are already ordered alphabetically. 

---
In reply to: [324450008](https://github.com/dotnet/machinelearning/pull/4205#discussion_r324450008) [](ancestors = 324450008)